app/javascript/controllers/rainy_day_controller.js

```diff
@@ -1,14 +1,13 @@
 import { Controller } from "@hotwired/stimulus";
+//  the Mountain Project map should change to reflect the new area 
+// and the information below should change to reflect the area I clicked.
 
 
 export default class extends Controller {
-  connect() {
-    console.log('Hello, rainy day controller!');
-  }
-
-  // Rebuild the old JQuery based functionality here
-};
-
+  static targets = ["area1", "area2", "link"]; // we now have a this.sourceTarget or a this.sourceTargets array
+  
+  initialize(watchedAreaSlug, options) {
```

Comment:
Let's change this method to `connect` instead of using the class initialization method. `connect` is a method the Stimulus controller will call once the DOM is ready so it's a safer method to put code within vs `initialize` here

Reply:
Will do!